src/main/java/uk/gov/hmcts/reform/sendletter/services/pdf/PdfCreator.java

```diff
@@ -34,15 +32,13 @@ public PdfCreator(DuplexPreparator duplexPreparator, IHtmlToPdfConverter convert
         return PdfMerger.mergeDocuments(docs);
     }
 
-    public byte[] createFromBase64Pdfs(List<String> base64encodedDocs) {
-        Asserts.notNull(base64encodedDocs, "base64encodedDocs");
+    public byte[] createFromBase64Pdfs(List<byte[]> base64decodedDocs) {
+        Asserts.notNull(base64decodedDocs, "base64decodedDocs");
```

Comment:
is any other test case covering this ? If not could we please add it

Reply:
yes. all the same unit/integration tests plus additional integration tests introduced when base64 is received but not a pdf. this method is basically simplifying the approach since decoding happens upon deserialisation of the incoming request rather than further down the line as before (which was here in this file).